scripts/dumpapp

```diff
@@ -58,6 +58,7 @@ def read_frames(sock):
     if code == b'1':
       if n > 0:
         sys.stdout.buffer.write(read_input(sock, n, 'stdout blob'))
+        sys.stdout.buffer.flush()
```

Comment:
Oops, @rightparen and I even discussed doing this and I wrong asserted it was not necessary.  I forgot about the streaming use cases :)